xblock/fields.py

```diff
@@ -465,7 +473,7 @@ def _calculate_unique_id(self, xblock):
         for the field in its given scope.
         """
         key = scope_key(self, xblock)
-        return hashlib.sha1(key).hexdigest()
+        return hashlib.sha1(key.encode('utf-8')).hexdigest()
```

Comment:
How does this work in Python 2? The old code must have had `key` as bytes, is it now Unicode?